test/internal/scorecard.go

```diff
@@ -79,3 +79,17 @@ func (tc TestContext) AddScorecardCustomPatchFile() error {
 	}
 	return nil
 }
+
+func (tc TestContext) AddScorecardDevImage() {
+	// Replace the scorecard dev image in config files.
```

Comment:
we are replacing `quay.io/operator-framework/scorecard-test` with any tag to use dev.
may the comment is not required.

Reply:
Done